tests/PHPStan/Analyser/nsrt/array-count-values.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace ArrayCountValues;
+
+use function PHPStan\Testing\assertType;
+
+$ints = array_count_values([1, 2, 2, 3]);
+
+assertType('array<1|2|3, int<1, max>>', $ints);
```

Comment:
could we get more precise?

```suggestion
assertType('non-empty-array<1|2|3, int<1, max>>', $ints);
```

Reply:
good shout, sorted

Reply:
We could be even be more precise with one of those
```
array{1: int<1, max>, 2: int<1, max>, 3: int<1, max>}
```

cause `non-empty-array<1|2|3, int<1, max>>` might report 1,2 and 3 offsets as "maybe missing".

Do you think it's worth it ?